examples/CRISP/remappings.txt

```diff
@@ -5,3 +5,6 @@ risc0/=lib/risc0-ethereum/contracts/src/
 @gnosis-guild/enclave=node_modules/@gnosis-guild/enclave
 @excubiae/contracts=node_modules/@excubiae/contracts
 solady/=node_modules/solady
+@semaphore-protocol/contracts=node_modules/@semaphore-protocol/contracts
+@zk-kit/lean-imt.sol=node_modules/@zk-kit/lean-imt.sol
```

Comment:
These remappings need trailing slashes to work within the docker environement. Will post a PR to this PR as suggestion.